src/olympia/scanners/tests/test_serializers.py

```diff
@@ -13,4 +15,7 @@ def test_serialize(self):
             'scanner': result.get_scanner_name(),
             'label': None,
             'results': result.results,
+            'created': result.created.strftime(
```

Comment:
I gather that it's magic built into DRF that applies this `strftime` to the `created`?

Reply:
Yeah, this is https://www.django-rest-framework.org/api-guide/fields/#datetimefield